cafebabel/static/css/styles.css

```diff
@@ -1610,6 +1673,9 @@ textarea#body {
     margin-left: 0.5rem;
     vertical-align: middle;
   }
+  .article-language-selector li.original-language a:before {
+    display: block;
```

Comment:
I'm not sure why you can't write it in https://github.com/cafebabel/cafebabel.com/pull/272/files#diff-6569206075db87419fc4b34e54f5eb4cR1645.
Just a quick reply will help ;)

Reply:
I don't want it to be displayed on mobile neither on click nor "always". And if I use `display:none` always and `display:block` on hover, I can't get any smooth transition. Was that what you meant?

Reply:
Right, makes sense now 👍